flask-server/avenv/lib/python3.12/site-packages/dotenv/cli.py

```
@@ -0,0 +1,205 @@
+    default=True,
+    help="Override variables from the environment file with those from the .env file.",
+)
+@click.argument('commandline', nargs=-1, type=click.UNPROCESSED)
+def run(ctx: click.Context, override: bool, commandline: List[str]) -> None:
+    """Run command with environment variables present."""
+    file = ctx.obj['FILE']
+    if not os.path.isfile(file):
+        raise click.BadParameter(
+            f'Invalid value for \'-f\' "{file}" does not exist.',
+            ctx=ctx
+        )
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)